exe/dirtree

```diff
@@ -4,6 +4,9 @@ require 'optparse'
 require 'json'
 require 'erb'
 require 'open-uri'
+require 'tempfile'
+
+default_file_name = 'output.html'
```

Comment:
this belongs to the screenshot feature and should not be confused with the `output` option, below there is more explanation 🔽